new-pipeline/style/dashboards.css

```diff
@@ -151,3 +151,10 @@
 .measure-headers th {
   text-align: right;
 }
+
+.portal-header-bar {
+  background-color: #EBEBEB;
+  margin-left: -15px;
+  margin-right: -15px;  
+  padding: 1px 15px 1px;
```

Comment:
I'm confused about this use of padding and margin. Could you explain to me its purpose?

Reply:
@chutten  The page has a default margin, so to override that I had to use margin and padding...hence margin gets a negative value, the padding was to increase the width as it looked a bit compact...Earlier I had used " :after setting content: " " and z-index=-1..but Georg had asked me to use margin and padding hence I made the above changes

Reply:
The branch now tracks moz/gh-pages....I'm not able to provide a live instance as somehow the github page hasn't updated,what could be the reason?